packages/eslint-config/package.json

```
@@ -22,26 +22,26 @@
   },
   "dependencies": {
     "@types/vfile-message": "^2.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.8.1",
-    "@typescript-eslint/parser": "^4.8.1",
-    "eslint": "^7.13.0",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-config-standard": "^16.0.2",
+    "@typescript-eslint/eslint-plugin": "^4.9.1",
+    "@typescript-eslint/parser": "^4.9.1",
+    "eslint": "^7.15.0",
+    "eslint-config-prettier": "7.0.0",
+    "eslint-config-standard": "16.0.2",
     "eslint-import-resolver-typescript": "^2.3.0",
     "eslint-import-resolver-webpack": "^0.13.0",
     "eslint-plugin-header": "^3.1.0",
     "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest-dom": "3.2.4",
+    "eslint-plugin-jest-dom": "3.6.3",
     "eslint-plugin-mdx": "^1.8.2",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-prettier": "3.2.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-sort-keys-fix": "^1.1.1",
-    "eslint-plugin-standard": "^4.1.0",
+    "eslint-plugin-standard": "5.0.0",
     "eslint-plugin-testing-library": "3.10.1",
-    "prettier": "^2.2.0",
+    "prettier": "^2.2.1",
```

```suggestion
    "prettier": "2.2.1",
```
I just realized I was getting a `yarn.lock` diff for `prettier`. This change just causes a different diff though. :)